frontend/src/components/utils/Balance.jsx

```diff
@@ -9,10 +9,11 @@ const Balance = ({ amount }) => {
     throw new Error("amount property should not be null");
   }
   let amountShow = convertToShow(amount);
+  let amountPrecise = formatPreciseNEAR(amount);
   return (
     <OverlayTrigger
       placement={"bottom"}
-      overlay={<Tooltip>{amount} yokto Ⓝ</Tooltip>}
+      overlay={<Tooltip>{amountPrecise} yokto Ⓝ</Tooltip>}
```

Comment:
It is "yocto" with "c" instead of "k", and let's remove the space between yocto and N: "yoctoⓃ"